### DISCUSSION TITLE
Removing default admin user

### DISCUSSION THREAD

**Discussion by AsaTyr2018** [mastodon/mastodon#25836] [Q&A] [answered]
Good day,

i installed mine via Azure Cloud. There is a default User present called "User".
it has the owner role and until i managed to change the defaul Mail Adress my mail account was spammed with "Mail Delivery" errors due to non existing default domain.

I can´t remove or block this account. If i try to block it. i get a "We're sorry, but something went wrong on our end." message.

Any Ideas?

**Comment by ryanmio** [comment, marked as answer]
I think a permission issue. Default admin can't usually be blocked via the UI. Try removing or altering the "owner" role directly in the database
